banano-quest/banano-quest/Utils/Networking.swift

```
@@ -0,0 +1,105 @@
+//
+//  Networking.swift
+//  banano-quest
+//
+//  Created by Pabel Nunez Landestoy on 6/25/18.
+//  Copyright © 2018 Michael O'Rourke. All rights reserved.
+//
+
+import Foundation
+import PocketEth
+import Pocket
+
+public class Networking {
+    static let nodeURL = URL.init(string: "https://node.url")
+    
+    public static func getQuestList() {
+
+        var params = [AnyHashable : Any]()
+        let count = getQuestListCount()
+        var index = count - 1
+
+        while index >= 0{
+            params["_tokenAddress"] = "tokenaddress"
+            params["_questIndex"] = index
+            
+            let query = createQuery(with: params)
+            
+            Pocket.getInstance(pocketNodeURL: nodeURL!).executeQuery(query: query, handler: { (response, error) in
+                if error != nil {
+                    print("")
+                }else{
+                    // TODO: parse response
+                    QuestDownloadParsing.parseDownload(dict: response!)
+                    print("")
+                }
+            })
+            index = index - 1
+        }
+        // Refresh current view
+        guard let activeVC = UIApplication.shared.delegate?.window??.rootViewController as? BananoQuestView else{
+            print("")
```

Fails silently, also let's do a completion handler.